aspnetcore/blazor/javascript-interoperability/call-dotnet-from-javascript.md

```diff
@@ -162,7 +162,25 @@ Call `DotNet.createJSStreamReference(streamReference)` to construct a JS stream
 To invoke an instance .NET method from JavaScript (JS):
 
 * Pass the .NET instance by reference to JS by wrapping the instance in a <xref:Microsoft.JSInterop.DotNetObjectReference> and calling <xref:Microsoft.JSInterop.DotNetObjectReference.Create%2A> on it.
-* Invoke a .NET instance method from JS using `invokeMethodAsync` or `invokeMethod` (Blazor WebAssembly only) from the passed <xref:Microsoft.JSInterop.DotNetObjectReference>. The .NET instance can also be passed as an argument when invoking other .NET methods from JS.
+* Invoke a .NET instance method from JS using `invokeMethodAsync` (*Recommended*) or `invokeMethod` (Blazor WebAssembly only) from the passed <xref:Microsoft.JSInterop.DotNetObjectReference>. Pass the identifier of the instance .NET method and any arguments. The .NET instance can also be passed as an argument when invoking other .NET methods from JS.
+
+  In the following example:
+
+  * The `{.NET METHOD ID}` placeholder is the .NET method identifier.
+  * The `{ARGUMENTS}` placeholder are optional, comma-separated arguments to pass to the method, each of which must be JSON-serializable.
+
+  ```javascript
+  DotNet.invokeMethodAsync('{.NET METHOD ID}', {ARGUMENTS});
```

Comment:
Hm... the `DotNet` object can only be used to invoke static methods, not instance methods. Did you intend for this example to be calling `invokeMethodAsync` on a `DotNetObjectReference`?

Reply:
For instance, the example later in the document using `dotNetHelper` is correct (with your fix, that is 🙂).